spec/models/profile_spec.rb

```diff
@@ -299,9 +299,39 @@
 
       expect(profile).to be_active
     end
+
+    context 'when the initiating_sp is the IRS' do
+      it 'logs an attempt event' do
+        sp = create(:service_provider, :irs)
+        profile = create(
+          :profile, user: user, active: false, fraud_review_pending: true,
+                    initiating_service_provider: sp
+        )
+        expect(profile.initiating_service_provider.irs_attempts_api_enabled?).to be_truthy
+
+        expect_any_instance_of(IrsAttemptsApi::Tracker).to receive(:fraud_review_adjudicated).
+          with(determination: 'activated')
+        profile.activate_after_passing_review
+      end
+    end
+
+    context 'when the initiating_sp is not the IRS' do
+      it 'does not log an attempt event' do
+        sp = create(:service_provider)
+        profile = create(
+          :profile, user: user, active: false, fraud_review_pending: true,
+                    initiating_service_provider: sp
+        )
+        expect(profile.initiating_service_provider.irs_attempts_api_enabled?).to be_falsey
+
+        expect_any_instance_of(IrsAttemptsApi::Tracker).not_to receive(:profile_activated)
+        profile.activate_after_passing_review
+      end
+    end
   end
 
   describe '#deactivate_for_fraud_review' do
+    # MAW: This is not this story.
```

Comment:
The _next_ method needs a spec written as well, but it needs a little refactoring and it's 10pm and so this is Tomorrow Matt problem.